frontend/src/components/SearchBar.tsx

```diff
@@ -32,17 +31,52 @@ const SearchBar: React.FC<SearchBarProps> = ({ onSearch }) => {
                 }}
                 value={searchTerm}
                 // updates the search bar's text as the user types into it
```

Comment:
seems like there's a certain width to click on search bar that initiates search. (I.e. when you click on the end of the search bar farther from the "Search Name or Title" text, it does not initiate the search)